Cargo.toml

```diff
@@ -11,15 +11,16 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [dependencies]
-wasm-bindgen = {version = "~0.2", features = ["serde-serialize"]}
+wasm-bindgen = { version = "~0.2", features = ["serde-serialize"] }
 serde = "~1.0"
 js-sys = "~0.3"
 pbkdf2 = { version = "0.3", default-features = false }
 hmac = "0.7"
 sha2 = "0.8"
-rand = { version = "0.6", features = ["wasm-bindgen"] }
+rand = { version = "~0.7.3", features = ["wasm-bindgen"] }
 serde_derive = "~1.0"
-recrypt = { version = "0.9.0", features = ["wasm"], default-features = false }
+recrypt = { version = "~0.11.0", features = ["wasm"], default-features = false }
+ironcore-search-helpers = "~0.1.0"
 
 [profile.dev]
 opt-level = 2 # Build deps with optimization, don't build ourselves with optimization
```

Comment:
Since you seem to be a moderate expert on this, is there changes we should make to this profile stuff to make it the same as our other repos? Maybe just adding `opt-level = 3` to the `profile.release` is all we should do?

Reply:
Release's opt-level is 3 by default, so it's somewhat redundant to have it there, but it possibly just helps us double-check that it's fully optimized. The only real change we make to release is turning on lto. 
I made a small change to what's there just to make it more similar to our other repositories.